src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs

```diff
@@ -661,6 +661,13 @@ internal string DecodeGuidAttribute(AttributeSyntax? nodeOpt, BindingDiagnosticB
             return guidString!;
         }
 
+        internal CollectionBuilderAttributeData DecodeCollectionBuilderAttribute()
+        {
+            var builderType = (TypeSymbol?)CommonConstructorArguments[0].ValueInternal;
+            var methodName = (string?)CommonConstructorArguments[1].ValueInternal;
```

Comment:
does some other part of the system ensure that it has at least two arguments?

Reply:
The caller verified this by checking the attribute matches the signature declared in `AttributeDescription.CollectionBuilderAttribute`.